modules/gitops-installing-argo-rollouts-cli-using-linux.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-installing-argo-rollouts-using-linux_{context}"]
+= Installing Argo Rollouts CLI using Linux
+
+For Linux, you can install Argo Rollouts CLI by performing the following steps.
+
+.Prerequisites
+
+* You have installed the {OCP} CLI (oc).
```

Comment:
Just making sure, are these all the prereqs? What about creating a RolloutManager custom resource?

Reply:
Yes, this was the only pre-req required and added in the draft document attached in the JIRA ticket for this PR.

Reply:
I can confirm RolloutManager resource is not required to install the CLI.